GooglePayment/src/main/java/com/braintreepayments/api/GooglePayment.java

```diff
@@ -77,6 +78,23 @@
      */
     public static void isReadyToPay(final BraintreeFragment fragment,
                                     final BraintreeResponseListener<Boolean> listener) {
+        isReadyToPay(fragment, null, listener);
+    }
+
+    /**
+     * Before starting the Google Payments flow, use this method to check whether the
+     * Google Payment API is supported and set up on the device. When the listener is called with
+     * {@code true}, show the Google Payments button. When it is called with {@code false}, display other
+     * checkout options.
+     *
+     * @param fragment {@link BraintreeFragment}
+     * @param request {@link ReadyForGooglePayRequest}
```

Comment:
Do we want to specify that this is optional (nullable)?

Reply:
Done ✔️